ql/src/test/queries/clientpositive/acid_direct_update_delete.q

```diff
@@ -18,4 +18,16 @@ DELETE FROM test_update_bucketed WHERE id IN ('2', '11', '10');
 UPDATE test_update_bucketed SET value='New value2' WHERE id IN ('2','18', '19');
 SELECT * FROM test_update_bucketed;
 
-DROP TABLE IF EXISTS test_update_bucketed;
\ No newline at end of file
+CREATE TABLE test_delete(id int) STORED AS ORC TBLPROPERTIES('transactional'='true');
+set mapreduce.job.reduces=7;
```

Comment:
no need to set # of reducers! please check the provided snippet

Reply:
Updated it.
I was testing few things and missed removing those lines. Thanks for finding it.